ui/handle_input.go

```diff
@@ -126,6 +126,46 @@ func (win *Window) HandleInputs() {
 		}
 	}
 
+	// CHANGE WALLET PASSWORD
+
+	if win.inputs.changePasswordDiag.Clicked(win.gtx) {
```

Comment:
```suggestion
	for win.inputs.changePasswordDiag.Clicked(win.gtx) {
```
So as to catch subsequent clicks